### ISSUE TITLE
Second level of lists hard to grasp

### ISSUE THREAD

**Issue by brotkrueml** [TYPO3-Documentation/render-guides#767]
The second level of list points are hard to grasp because of the bullet type:

![image](https://github.com/user-attachments/assets/5b0819ff-cee0-41b3-a77d-a8b302c6e098)

These bullet points should be better visible, also to enhance accessibility.
